worlds/noita/Locations.py

```diff
@@ -0,0 +1,213 @@
+# Locations are specific points that you would obtain an item at.
+from enum import IntEnum
+from typing import Dict, NamedTuple, Optional
+
+from BaseClasses import Location
+
+
+class NoitaLocation(Location):
+    game: str = "Noita"
+
+
+class LocationData(NamedTuple):
+    id: int
+    flag: int = 0
+    ltype: Optional[str] = ""
+
+
+class LocationFlag(IntEnum):
+    none = 0
+    main_path = 1
+    side_path = 2
+    main_world = 3
+    parallel_worlds = 4
+
+
+# Mapping of items in each region.
+# Only the first Hidden Chest and Pedestal are mapped here, the others are created in Regions.
+# ltype key: "chest" = Hidden Chests, "pedestal" = Pedestals, "boss" = Boss, "orb" = Orb.
+# 110501-112800
```

Comment:
We might want to revisit this to close the large gaps , we have ~669 locations, not 2300 locations

Reply:
Yeah... I've been wanting to put off doing this, but probably will after the sync today or tomorrow at some point.